content/integrations/awsrds.md

```diff
@@ -70,7 +70,7 @@ The command output will include two parts: a ciphertext blob followed by the key
 
 ##### Create your Lambda function
 
-12. From the Lambda Management Console, create a new Lambda Function. NOTE, your Lambda function must be in the same region as the KMS key you created.
+12. From the Lambda Management Console, create a new Lambda Function. **Your Lambda function MUST be in the same region as the KMS key you created.**
```

Comment:
Not sure we need both bold AND a MUST. Does this point need to be that severely made?